tests/src/chanjo2/endpoints/test_coverage.py

```diff
@@ -163,23 +163,23 @@ def test_d4_intervals_coverage(
 
 
 @pytest.mark.parametrize("build", Builds.get_enum_values())
-def test_samples_gene_coverage_hgnc_symbols(
+def test_sample_gene_coverage_hgnc_symbols(
     build: str,
     demo_client: TestClient,
     endpoints: Type,
     genomic_ids_per_build: Dict[str, List],
 ):
     """Test the function that returns the coverage over multiple genes of a sample when a list of HGNC symbols is provided."""
 
-    # GIVING a sample coverage query containing HGNC gene symbols
+    # GIVING a sample gene coverage query containing HGNC gene symbols
```

Comment:
```suggestion
    # GIVEN a sample gene coverage query containing HGNC gene symbols
```
Fix for all occurrences